subsys/bluetooth/host/Kconfig

```diff
@@ -753,6 +753,17 @@ config BT_DEBUG_SERVICE
 
 endif # BT_DEBUG
 
+config BT_LOG_SNIFFER_INFO
+	bool "Bluetooth log information for sniffer"
```

Comment:
Wouldn't it make sense to add `depends on BT_TESTING` here? `BT_TESTING` is intended to make clear which options should not end up in final product configurations.

Reply:
I'm unsure. BT_TESTING is mostly used for options that changes the behavior, while this only enables logging of certain information.

Looking over the options that we warn on, but don't have depends on BT_TESTING we have the ones that just logs information, but logs security info:
CONFIG_BT_DEBUG_SMP, CONFIG_BT_DEBUG_KEYS, BT_LOG_SNIFFER_INFO.
And the ones that changes behavior:
CONFIG_BT_FIXED_PASSKEY, CONFIG_BT_USE_DEBUG_KEYS, CONFIG_BT_STORE_DEBUG_KEYS